docs/modules/edged.md

```diff
@@ -0,0 +1,61 @@
+# EdgeD
+
+## Overview
+
+EdgeD is an edge node module which manages the POD lifecycle. With help of the POD specifications provided as JSON from Cloud via MetaManager, it ensures the containers for the POD's (with containers as described in POD spec) are launched over the edge node, are running and healthy.
+
+EdgeD interfaces with MetaManager module to receive all the configuration calls. It responds to MetaManager module with all config results and queries.
+
+Docker container runtime is used for container and image management. It is the default option for EdgeD. It is a configurable option in config.yaml.
+
+Following functionalities are performed by edgeD :-
+
+- POD Launch, Deletion & Modification
+- ConfigMap Addition, Deletion & Modification
+- Secret Addition, Deletion & Modification
+- Probing Containers for Liveliness and Readiness
+
+## POD Launch, Deletion & Modification
+
+EdgeD supports Adding (Launching), Modification and Deletion of POD's. These requests are sent from Cloud, whenever respective "Kubectl" pod commands are execute over its terminal. The specs are defined in yaml format which is similar to that of Kubernetes. Those spec's can also have Volume and Network specs.
```

Comment:
POD's -> pods.

Reply:
The specs are defined in yaml format which is similar to that of Kubernetes.
--> Why do we need to specify this ? Is it not the same pod yaml definition as in Kubernetes ? Ideally we should not specify these details. Rather say that we support the deployment of Kubernetes pods and give a link to K8S docs if needed.

 Those spec's can also have Volume and Network specs.
--> Why do we need to specify this ? Pod specs support many constructs. Are we saying we support only Volume ? Also what is Network spec here , does it mean NetworkPolicy ?  I feel we should not explain Kubernetes concepts here like pods , volumes etc. Let's